.github/workflows/nia-e2e.yaml

```
@@ -21,44 +21,88 @@ jobs:
     - name: Setup Anbox Cloud
       uses: canonical/anbox-cloud-github-action@dc99b94280cfcf44a86b62aeac25ba34f14725e0
       with:
-        channel: latest/stable
+        channel: 1.28/edge/amc-connect
 
     - name: Tune Anbox Cloud
       run: |
         amc config set container.security_updates false
 
+    - name: Setup remote amc
+      env:
+        REMOTE_AMC_URL: ${{ vars.REMOTE_AMC_URL }}
+        REMOTE_CLIENT_CERT: ${{ secrets.REMOTE_CLIENT_CERT }}
+        REMOTE_CLIENT_KEY: ${{ secrets.REMOTE_CLIENT_KEY }}
+      run: |
+        set -x
+
+        url=${REMOTE_AMC_URL}
+        if [ -z "${url}" ]; then
+            ip=$(hostname -I | awk '{print $1}')
+            url="https://${ip}:8444"
+            amc config trust add "$HOME"/snap/anbox-cloud-appliance/current/client/client.crt
+        fi
+
+        if [ -n "${REMOTE_CLIENT_CERT}" ]; then
+          echo "${REMOTE_CLIENT_CERT}" >> "$HOME"/snap/anbox-cloud-appliance/current/client/remote_client.crt
+          amc config trust add "$HOME"/snap/anbox-cloud-appliance/current/client/remote_client.crt
```

Lets create an identity instead of using the old certificate path


```suggestion
          echo "$REMOTE_CLIENT_CERT" | amc auth identity create tls/test-user -
```

That also lets you avoid writing the cert anywhere.